src/wallet/wallet2.cpp

```diff
@@ -2772,8 +2780,8 @@ bool wallet2::load_keys(const std::string& keys_file_name, const epee::wipeable_
     GET_FIELD_FROM_JSON_RETURN_ON_ERROR(json, store_tx_keys, int, Int, false, true);
     GET_FIELD_FROM_JSON_RETURN_ON_ERROR(json, store_tx_info, int, Int, false, true);
     m_store_tx_info = ((field_store_tx_keys != 0) || (field_store_tx_info != 0));
-    GET_FIELD_FROM_JSON_RETURN_ON_ERROR(json, default_mixin, unsigned int, Uint, false, 0);
-    m_default_mixin = field_default_mixin;
+    GET_FIELD_FROM_JSON_RETURN_ON_ERROR(json, default_ring_size, unsigned int, Uint, false, 0);
+    m_default_ring_size = field_default_ring_size;
```

Comment:
I don't think this will be a problem because the wallet keys file in the current Aeon release is not using this JSON format. I.e. there's no `m_default_mixin` setting that could've possibly been set previously using the current 0.9.14 binary.

Reply:
Oh wow, that's old :)